src/components/SelectedMovie/SelectedMovie.css

```diff
@@ -49,11 +43,78 @@
       display: flex;
       flex-direction: row;
       justify-content: space-between;
+      /* margin-left: 1em; */
   }
 
-  /* h3 {
-      font-size: 5em;
-  } */
+  .movie-title {
+    font-size: 4em;
+    margin: 0 0 0 0;
+    display: flex;
+    justify-content: center;
+    margin-bottom: .5em;
+  }
+
+.sidebar {
+    font-size: .75em;
+    border: 5px dotted yellow;
+    margin-left: 2em;
+    width: 12vw;
+    height: 52vh;
+
+}
+
+.overview {
+    border: 5px dotted yellow;
+    height: 60vh;
+    width: 50vw;
+    margin-left: 2em;
+}
+
+
+button{
+    width:100%;
+    height:80px;
+    position:relative;
+    background:rgba(255, 255, 255, .27);
+    text-transform:uppercase;
+    color:white;
+    font-weight:700;
+    letter-spacing:1px;
+    border:none;
+    font-size:15px;
+    outline:none;
+    font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+    cursor: pointer;
+  }
+
+  .border{
+    position:absolute;
+    background:none;
+    transition:all .5s ease-in-out;
+}
+
+#first>.border:nth-of-type(1){
+  top:0;
+  left:0;
+  border-left:1px solid white;
+  border-top:1px solid white;
+  width:30px;
+  height:30px;
+}
+
+#first>.border:nth-of-type(2){
```

Comment:
oooo! I have never used this kind of selector before. I will definitely be doing some more research on this later!